kuka_lbr_iiwa_support/package.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<package>
+  <name>kuka_lbr_iiwa_support</name>
+  <version>0.0.0</version>
+  <description>
+    <p>
+      ROS-Industrial support for the KUKA LBR IIWA (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for KUKA LBR IIWA manipulators. This currently includes the base model.
+    </p>
+    <p>
+      Joint limits and max joint velocities are based on the information in
+      the KUKA data sheets.  All URDFs / XACROs are based on the
```

Comment:
Also: double space (between 'sheets.' and 'All ..').

Also: lower case 'urdf' and 'xacros'